### PR TITLE
Adding php

### PR DESCRIPTION
**Pull request by Heccubernny**
Adding PHP programming language details according to the issue created by  [ericspring08](https://github.com/ericspring08)
Contributing to the vision of this project and resolving the issue - [#1305578920](https://github.com/ericspring08/Awesome-Programming-Languages/issues/9#issue-1305578920). Adding the PHP programming language

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -27,6 +27,8 @@ Please submit a issue [here](https://github.com/ericspring08/Awesome-Programming
 - [Julia](Languages/Julia.md) - Julia is a dynamic, high-performance, compiler based programming language that is used to perform operations in scientific computing. Similar to R Programming Language, Julia is used for statistical computations and data analysis. Julia was built mainly because of its speed in programming, it has much faster execution as compared to Python and R.
 - [Lua](Languages/Lua.md) - Lua from Portuguese, meaning "moon" is a lightweight, high-level, multi-paradigm programming language designed primarily for embedded use in applications.
   Lua is cross-platform, since the interpreter of compiled bytecode is written in ANSI C, and Lua has a relatively simple C API to embed it into applications.
+  [Php](Languages/Php.md) - HP is a popular general-purpose scripting language that is especially suited to web development.
+Fast, flexible and pragmatic, PHP powers everything from your blog to the most popular websites in the world.
```
Comment (ericspring08) [inline README.md:31]:
```suggestion
Fast, flexible and pragmatic, PHP powers everything from your blog to the most popular websites in the world. Fast, flexible and pragmatic, PHP powers everything from your blog to the most popular websites in the world.
```

File: Languages/Php.md [hunk 2]
```diff
@@ -0,0 +1,16 @@
+# Php
+- **Description** - PHP is a popular general-purpose scripting language that is especially suited to web development.
```
Comment (ericspring08) [inline Languages/Php.md:2]:
```suggestion
- **Description** - PHP is a popular general-purpose scripting language that is especially suited to web development. Fast, flexible and pragmatic, PHP powers everything from your blog to the most popular websites in the world.
```

File: Languages/Php.md [hunk 3]
```diff
@@ -0,0 +1,16 @@
+# Php
+- **Description** - PHP is a popular general-purpose scripting language that is especially suited to web development.
+Fast, flexible and pragmatic, PHP powers everything from your blog to the most popular websites in the world.
```
Comment (ericspring08) [inline Languages/Php.md:3]:
```suggestion


```

File: README.md (hunk 1 above)
Comment (ericspring08) [inline README.md:31]:
```suggestion

```

File: README.md [hunk 4]
```diff
@@ -27,6 +27,8 @@ Please submit a issue [here](https://github.com/ericspring08/Awesome-Programming
 - [Julia](Languages/Julia.md) - Julia is a dynamic, high-performance, compiler based programming language that is used to perform operations in scientific computing. Similar to R Programming Language, Julia is used for statistical computations and data analysis. Julia was built mainly because of its speed in programming, it has much faster execution as compared to Python and R.
 - [Lua](Languages/Lua.md) - Lua from Portuguese, meaning "moon" is a lightweight, high-level, multi-paradigm programming language designed primarily for embedded use in applications.
   Lua is cross-platform, since the interpreter of compiled bytecode is written in ANSI C, and Lua has a relatively simple C API to embed it into applications.
+  [Php](Languages/Php.md) - HP is a popular general-purpose scripting language that is especially suited to web development.
```
Comment (ericspring08) [inline README.md:30]:
```suggestion
  [Php](Languages/Php.md) - HP is a popular general-purpose scripting language that is especially suited to web development. Fast, flexible and pragmatic, PHP powers everything from your blog to the most popular websites in the world.
```

**Comment by Heccubernny** [comment]
will work on it thanks

**Comment by Heccubernny** [comment]
Fix all issues. Thank you

**Comment by ericspring08** [comment]
Thank you for contributing!

**Pull request merged**